### PR TITLE
:sparkles: Added smooth transition to tooltip (#268)

### PR DESCRIPTION
**Pull request by HappyMakadiyaS**
Fixes [#268](https://github.com/SimformSolutionsPvtLtd/flutter_showcaseview/issues/268)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,9 +63,15 @@ Some more optional parameters
 Showcase(
   key: _two,
   title: 'Profile',
-  description: 'Click here to go to your Profile',
-  disableAnimation: true,
-  shapeBorder: CircleBorder(),
+description: '
```
Comment (vatsaltanna) [inline README.md:66]: format has been changed, @HappyMakadiyaS can you please update this.

File: lib/src/tooltip_widget.dart [hunk 2]
```diff
@@ -362,35 +452,44 @@ class _ToolTipWidgetState extends State<ToolTipWidget>
...
+                          child: MeasureSize(
+                              onSizeChange: (size) {
+                                setState(() {
+                                  var tempPos = position;
+                                  tempPos = Offset(position!.dx,
+                                      position!.dy + size!.height);
+                                  position = tempPos;
+                                });
+                              },
+                              child: widget.container),
```
Comment (vatsaltanna) [inline lib/src/tooltip_widget.dart:491]: please add comma before bracket.

File: README.md [hunk 3]
```diff
@@ -63,9 +63,15 @@ Some more optional parameters
 Showcase(
```
Comment (vatsaltanna) [inline README.md:63]: please update gif for the example too

File: README.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ A Flutter package allows you to Showcase/Highlight your widgets step by step.
 
 ## Preview
 
-![The example app running in Android](https://github.com/simformsolutions/flutter_showcaseview/blob/master/preview/showcaseview.gif)
+![The example app running in Android](preview/showcaseview.gif)
```
Comment (vatsaltanna) [inline README.md:12]: please use github url only as we not sure if this will work in pub or not.

**Review by vatsaltanna-simformsolutions — APPROVED** [review]

**Pull request merged**